index.html

```diff
@@ -352,21 +351,20 @@ <h3>Ecosystem Overview</h3>
           </dd>
           <dt><a>verifier</a></dt>
           <dd>
-A role an <a>entity</a> might perform by requesting and receiving a
-<a>verifiable presentation</a> that proves the <a>holder</a> possesses the
-required <a>verifiable credentials</a> with certain characteristics. Example
-verifiers include employers, security personnel, and websites.
+A role an <a>entity</a> might perform by receiving one or more
+<a>verifiable presentations</a> for processing. Example verifiers include
```

Comment:
verifiable presentations are optional not mandatory. So reword to

A role an <a>entity</a> might perform by receiving one or more <a>verifiable credentials</a>,
optionally inside a <a>verifiable presentation</a>, for processing.

Reply:
Changes made in [52d48bc](https://github.com/w3c/vc-data-model/pull/508/commits/52d48bc92cf0b371d91a42c7ac8a3394bfba51b4)